src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -672,13 +672,40 @@
     (assoc q :fuel max-fuel)
     q))
 
+(defn extract-values
```

Comment:
This pattern is repeated a lot in this pr. Perhaps we could generalize it with something like this:

```clojure
(defn get-named
  [jsonld nme]
  (or (get jsonld nme)
      (get jsonld (keyword nme))))

(let [values (get-named jsonld "values")]
  ...)
```

Reply:
Resolved in [5528eef](https://github.com/fluree/db/pull/983/commits/5528eef213b5e65bb88e7c9cbdd1351ad7c63dc5)